pandas/tests/frame/methods/test_rank.py

```diff
@@ -26,8 +28,10 @@ def method(self, request):
         """
         return request.param
 
+    @td.skip_if_no_scipy
     def test_rank(self, float_frame):
-        rankdata = pytest.importorskip("scipy.stats.rankdata")
```

Comment:
Were these ever running before? I guess this isn't directly importable

Reply:
AFAICT no.  im working on a branch now to reduce how many skips we have so we can keep a closer eye on them

Reply:
Great!